### ISSUE TITLE
something-something-V1 16 segments results  reproduction

### ISSUE THREAD

**Issue by yzfly** [arunos728/MotionSqueeze#14]
hello, I try to reproduce the something-V1 16 segments results, but only got Testing Results: Prec@1 51.1. 

The training settings:

-num_segments 16 --mode 1 --gd 200 \
--lr 0.01 --lr_steps 20 30  --epochs 40 -b 32 -i 1 -j 16 --dropout 0.5

Any suggestions?

**Comment by XiaoXueHou** [comment]
@yzfly 朋友你好！冒昧打扰一下，我最近也在复现这篇论文的工作，但是按照作者提供的Conda环境，我一直无法成功安装spatial-correlation-sampler库，尝试了很多很多遍都没有成功。所以想请教你一下，你对于这篇论文工作的复现时按照作者提供的CUDA9.0，GCC5.4，Pytorch1.0.1等CondaList.txt上面的环境嘛？或者你是在什么样的环境下成功复现这篇工作的？期待你的回复，万分感谢🙏

**Comment by yzfly** [comment]
> @yzfly 朋友你好！冒昧打扰一下，我最近也在复现这篇论文的工作，但是按照作者提供的Conda环境，我一直无法成功安装spatial-correlation-sampler库，尝试了很多很多遍都没有成功。所以想请教你一下，你对于这篇论文工作的复现时按照作者提供的CUDA9.0，GCC5.4，Pytorch1.0.1等CondaList.txt上面的环境嘛？或者你是在什么样的环境下成功复现这篇工作的？期待你的回复，万分感谢

pytorch1.6    CUDA10.2  spatial-correlation-sampler库的安装参照官方 github 库就行
